CMakeLists.txt

```diff
@@ -161,32 +163,34 @@ ELSE (GIT_BINARY)
 	ENDIF (NOT GIT_FOUND)
 ENDIF (GIT_BINARY)
 
-IF (PREFIX)
+# PREFIX has no sence in case of win32
+IF (PREFIX AND NOT WIN32)
 	ADD_DEFINITIONS(-DPREFIX=${PREFIX})
-ENDIF (PREFIX)
+ENDIF (PREFIX AND NOT WIN32)
 
 # Set build types and display info
-	MESSAGE(STATUS "DYNAMIC_LINKING               : ${DYNAMIC_LINKING}\n")
 IF(RELEASE)
 	MESSAGE(STATUS "CMAKE_INSTALL_PREFIX          : ${CMAKE_INSTALL_PREFIX}")
 	MESSAGE(STATUS "BIN_PREFIX                    : ${BIN_PREFIX}")
 	MESSAGE(STATUS "DATA_PREFIX                   : ${DATA_PREFIX}")
+    IF(LOCALIZE)
+        MESSAGE(STATUS "LOCALE_PATH                   : ${LOCALE_DIR}")
+    ENDIF(LOCALIZE)
 	MESSAGE(STATUS "DESKTOP_ENTRY_PATH            : ${DESKTOP_ENTRY_PATH}")
 	MESSAGE(STATUS "PIXMAPS_ENTRY_PATH            : ${PIXMAPS_ENTRY_PATH}")
 	MESSAGE(STATUS "PIXMAPS_UNITY_ENTRY_PATH      : ${PIXMAPS_UNITY_ENTRY_PATH}")
 	MESSAGE(STATUS "MANPAGE_ENTRY_PATH            : ${MANPAGE_ENTRY_PATH}\n")
 	SET(CMAKE_BUILD_TYPE RELEASE)
+	ADD_DEFINITIONS(-DRELEASE)
 	# FIXME: Removed -Werror for a while. Currently it is broken in upstream.
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Os -std=c++11 -MMD -Wall -Wextra")
 	IF(MINGW)
 		# Workaround for: http://ehc.ac/p/mingw/bugs/2250/
 		ADD_DEFINITIONS(-D__NO_INLINE__)
 	ENDIF(MINGW)
-	IF(NOT WIN32 AND NOT PREFIX)
-		# PREFIX has no sence in case of win32
-		ADD_DEFINITIONS(-DDATA_DIR_PREFIX=${CMAKE_INSTALL_PREFIX})
-	ENDIF(NOT WIN32 AND NOT PREFIX)
-	ADD_DEFINITIONS(-DRELEASE)
+#	IF(NOT WIN32 AND NOT PREFIX)
```

Comment:
Clion always outputs to a folder for the build configuration, if I set
the output location to my cataclysm directory, it would output to
`Cataclysm-DDA\Debug`

On 7/27/15, Vlasov Vitaly <notifications@github.com> wrote:
>>  	# FIXME: Removed -Werror for a while. Currently it is broken in
>> upstream.
>>  	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Os -std=c++11 -MMD -Wall
>> -Wextra")
>>  	IF(MINGW)
>>  		# Workaround for: http://ehc.ac/p/mingw/bugs/2250/
>>  		ADD_DEFINITIONS(-D__NO_INLINE__)
>>  	ENDIF(MINGW)
>> -	IF(NOT WIN32 AND NOT PREFIX)
>> -		# PREFIX has no sence in case of win32
>> -		ADD_DEFINITIONS(-DDATA_DIR_PREFIX=${CMAKE_INSTALL_PREFIX})
>> -	ENDIF(NOT WIN32 AND NOT PREFIX)
>> -	ADD_DEFINITIONS(-DRELEASE)
>> +#	IF(NOT WIN32 AND NOT PREFIX)
>
> Ok, fallback to old PREFIX control.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/CleverRaven/Cataclysm-DDA/pull/13047/files#r35529282